README.md

```diff
@@ -63,6 +63,9 @@ The following dependencies are required to build it:
 - meson
 - go-md2man
 - systemd
+- a c compiler. gcc is known to work
```

Comment:
I'd say that mentioning gcc is not needed because Go depends on it, right?

Reply:
Done